src/benchmark/processes.py

```diff
@@ -73,21 +74,116 @@ def _fill_command_line(self):
         pass
 
     def __print_error(self):
-        out = self._my_output
+        out = self._output
         is_error = False
         for line in out:
             if line.rfind('ERROR! :') != -1:
                 is_error = True
-                self.__my_log.error('    {0}'.format(line[8:]))
+                self.__log.error('    {0}'.format(line[8:]))
                 continue
             if is_error:
-                self.__my_log.error('    {0}'.format(line))
+                self.__log.error('    {0}'.format(line))
 
 
 class OpenVINOProcess(ProcessHandler, ABC):
     def __init__(self, test, executor, log):
         super().__init__(test, executor, log)
 
+    @staticmethod
+    def create_process(test, executor, log):
+        mode = test.dep_parameters.mode.lower()
+        if mode == 'sync':
+            return SyncOpenVINOProcess(test, executor, log)
+        elif mode == 'async':
+            return AsyncOpenVINOProcess(test, executor, log)
+        elif mode == 'ovbenchmarkpython':
+            return OpenVINOBenchmarkPythonProcess(test, executor, log)
+
+
+class OpenVINOBenchmarkPythonProcess(OpenVINOProcess):
+    def __init__(self, test, executor, log):
+        super().__init__(test, executor, log)
+
+    @staticmethod
+    def __add_perf_hint_for_cmd_line(command_line, perf_hint):
+        hint = perf_hint.lower()
+        if hint in ('latency', 'throughput'):
+            return f'{command_line} -hint {hint}'
+        return command_line
+
+    @staticmethod
+    def __add_extension_for_cmd_line(command_line, extension, device):
+        if 'GPU' in device:
+            return '{0} -c {1}'.format(command_line, extension)
```

Comment:
please, change all formatting to f-string, since it is recommended way starting from python 3.6